ethereum/contracts/ETHApp.sol

```diff
@@ -74,16 +75,17 @@ contract ETHApp {
         emit Unlocked(_sender, _recipient, _amount);
     }
 
-    function encodeCall(address _sender, bytes32 _recipient, uint256 _amount)
-        private
-        pure
-        returns (bytes memory)
-    {
+    // SCALE-encode payload
+    function encodeCall(
+        address _sender,
+        bytes32 _recipient,
+        uint256 _amount
+    ) private pure returns (bytes memory) {
         return
             abi.encodePacked(
                 MINT_CALL,
                 _sender,
-                byte(0x00), // Encode recipient as MultiAddress::Id
+                bytes(0x00), // Encode recipient as MultiAddress::Id
```

Comment:
Can we leave it as `byte`? I made it byte since according to the solidity docs, `byte` is an alias for `bytes1` (There isn't a single byte type in Solidity, as a fixed-length byte array of length 1 is equal to a single byte).

Reply:
Yes, `byte` is more efficient than `bytes` (bytes is a dynamic array, which encoded needs a length prefix). See https://docs.soliditylang.org/en/v0.5.3/types.html#fixed-size-byte-arrays

Reply:
Changed it back to `byte` in https://github.com/Snowfork/polkadot-ethereum/pull/266/commits/7ef0b553f98b69069fb5b4f00f21f9fe24b147c7